plugins/module_utils/github.py

```diff
@@ -794,6 +796,10 @@ def _manage_org_team(
                     parent=target.get('parent'),
                     maintainers=target.get('maintainer', [])
                 )
+                if current is None:
+                    self.save_error(f"Unable to create team: {slug} / {target} with "
+                                    f"maintainers : {', '.join(target.get('maintainer', []))} "
+                                    "(all maintainers must be completely onboarded)")
```

Comment:
is GH failing to create team where maintainer hasn't accepted the org invite yet?

Reply:
Yes